relay-server/src/actors/spooler/mod.rs

```diff
@@ -209,8 +211,12 @@ impl InMemory {
             count += current_count;
             self.used_memory -= current_size;
         }
+        self.envelope_count -= count as isize;
```

Comment:
What's the type of `count`? If it's `u64`, it may not fit in `isize`. Should we try the casting instead (comment below)?

Reply:
It has to be _very_ large to not fit into `isize`, but changed to `usize` and `saturating_sub` regardless.